common/src/test/java/tachyon/security/authentication/AuthenticationFactoryTest.java

```diff
@@ -131,7 +121,9 @@ public void simpleAuthenticationTest() throws Exception {
     startServerThread(mTachyonConf);
 
     // when connecting, authentication happens. It is a no-op in Simple mode.
-    TTransport client = createClient(mTachyonConf, "anyone", "whatever");
+    TTransport client =
+        PlainSaslHelper.getPlainClientTransport("anyone", "whatever",
+            AuthenticationFactory.createTSocket(mServerAddress));
```

Comment:
`AuthenticationFactory.createTSocket(mServerAddress)` is called multiple times to create a `TSocket`  in different test cases. Is it possible to make it a member variable, instead of creating it each time?